### PR TITLE
Fix mobile inbox requests

### PR DESCRIPTION
**Pull request by sanderpick**
- On mobile, the inbox requests would fail if the sender did not have a session for the recipent's cafe
- Updates service handling to https://github.com/libp2p/go-libp2p-kad-dht/releases/tag/v0.1.1
- Force (permanently) adds the textile bootstap peers to the peerstore when the node comes online in an effort to kill the "no peer in table" errors

### CODE REVIEW

**Comment by sanderpick** [comment]
This can take some eyes, but I've been doing some digging to find why dials sometimes fail.

File: core/main.go [hunk 1]
```diff
@@ -376,6 +377,15 @@ func (t *Textile) Start() error {
 		}
 		log.Info("node is online")
 
+		// ensure the peer table is not epty by adding our bootstaps
```
Comment (carsonfarmer) [inline core/main.go]: minor typo nit here

File: core/main.go [hunk 2]
```diff
@@ -376,6 +377,15 @@ func (t *Textile) Start() error {
...
+		boots, err := config.TextileBootstrapPeers()
+		if err != nil {
+			log.Errorf(err.Error())
+		}
+		for _, p := range boots {
```
Comment (carsonfarmer) [inline core/main.go]: nice!

File: core/main_test.go [hunk 3]
```diff
@@ -55,6 +55,8 @@ func TestSetLogLevel(t *testing.T) {
 	logLevel := &pb.LogLevel{Systems: map[string]pb.LogLevel_Level{
 		"tex-core":      pb.LogLevel_DEBUG,
 		"tex-datastore": pb.LogLevel_INFO,
+		"tex-service":   pb.LogLevel_DEBUG,
+		//"swarm2":        pb.LogLevel_DEBUG,
```
Comment (carsonfarmer) [inline core/main_test.go]: is this commented out on purpose?

Comment (sanderpick) [reply core/main_test.go]: deleted

File: core/main.go (hunk 1 above)
Comment (sanderpick) [reply core/main.go]: thanks, updated!

**Comment by sanderpick** [comment]
Related to https://github.com/textileio/go-textile/issues/871, I added a `textile publish` command to this PR. I want to see if the remote receives pubsub messages during this "flaky downtime".

**Comment by sanderpick** [comment]
e.g.,

```
echo "boom" | textile publish /textile/cafe/1.0.0/12D3KooWSsM117bNw6yu1auMfNqeu59578Bct5V4S9fWxavogrsw
```

The remote will not recognize the message and safely bail. However, we'll be able to watch its logs and determine if it was received or not.

**Pull request merged**